source/user-manual/reference/ossec-conf/wodle-docker.rst

```diff
@@ -0,0 +1,94 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _wodle_docker:
+
+wodle name="docker-listener"
+============================
+
+.. versionadded:: 3.7.0
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<wodle name="docker-listener">
+		</wodle>
+
+Configuration options of the Docker wodle.
+
+Options
+-------
+
+- `interval`_
+- `attempts`_
+- `run_on_start`_
+- `disabled`_
+
++----------------------+-----------------------------+
+| Options              | Allowed values              |
++======================+=============================+
+| `interval`_          | A positive number           |
++----------------------+-----------------------------+
+| `attempts`_          | A positive number           |
++----------------------+-----------------------------+
+| `run_on_start`_      | yes, no                     |
++----------------------+-----------------------------+
+| `disabled`_          | yes, no                     |
++----------------------+-----------------------------+
+
+interval
+^^^^^^^^
+
+Waiting time to rerun the wodle in case it fails.
+
++--------------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| **Default value**  | 1h                                                                                                                                       |
```

Comment:
The default interval is **10 minutes**.

https://github.com/wazuh/wazuh/blob/238e145b722968933174a5d1ec08ecb24212063b/src/wazuh_modules/wm_docker.h#L19